src/main/java/core/basesyntax/strategy/impl/ReturnOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.strategy.OperationHandler;
+import java.util.Map;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void apply(String fruit, int quantity, Map<String, Integer> storage) {
+        storage.put(fruit, storage.getOrDefault(fruit, 0) + quantity);

```

Comment:
Consider checking if the storage has enough quantity before performing the return operation. This is to prevent potential issues that may arise when the quantity in the storage is less than the return quantity.